### PR TITLE
Minor-ish documentation additions

### PR DESCRIPTION
**Pull request by mkoistinen**
@evildmp please review

### CODE REVIEW

**Comment by mkoistinen** [comment]
Rebased

File: docs/introduction/installation.rst [hunk 1]
```diff
@@ -47,7 +47,12 @@ listed in ``INSTALLED_APPS``, *after* ``'cms'``.
 .. note::
    If you are using Django 1.6, add ``south`` to  ``INSTALLED_APPS``.
 
-And make sure that your other settings conform to::
+
+Aldryn Boilerplates
+===================
+
+Make sure you follow the installation directions for `Aldryn Boilerplates <https://github.com/aldryn/aldryn-boilerplates>`_.
```
Comment (evildmp) [inline docs/introduction/installation.rst]: Can we keep this to 100 columns?

File: docs/introduction/installation.rst [hunk 2]
```diff
@@ -83,12 +88,30 @@ Now set the name of the boilerplate you want to use in your project::
    ``/boilerplates`` directory.
 
 
+Filer
+=====
+
+Aldryn Events also depends on Filer, be sure to follow 
+`Filer's installation instructions <http://django-filer.readthedocs.org/en/latest/installation.html>`_.
+To get up and running quickly, make sure you adapt your settings to include the
+`filer.thumbnail_processors.scale_and_crop_with_subject_location`
```
Comment (evildmp) [inline docs/introduction/installation.rst]: Should be double-backticks around literals.

**Comment by evildmp** [comment]
Looks good.

**Pull request merged**